scripts/webknossos_annotation.py

```diff
@@ -0,0 +1,303 @@
+    dic_JW = {0:0, 1:1, 2:2, 3:3, 4:4, 6:5, 7:6, 8:7, 9:8}
+
+    # Prepare chunking options
+    opt = {
+        'chunks': [1, 1] + [chunk, chunk],
+        'dimension_separator': r'/',
+        'order': 'F',
+        'dtype': 'uint8',
+        'fill_value': None,
+        'compressor': make_compressor(compressor, **compressor_opt),
+    }
+    print(opt)
+
+
+    nblevel = 9
+    # Write each level
+    for level in range(nblevel):
+        omz_res = omz_data[level]
+        size = omz_res.shape[-2:]
+        shape = [1, 20] + [i for i in size] 
+        
+        wkw_dataset_path = os.path.join(wkw_dir, get_mask_name(level))
+        wkw_dataset = wkw.Dataset.open(wkw_dataset_path)
+
+        omz.create_dataset(f'{level}', shape=shape, **opt)
+        array = omz[f'{level}']
+
+        # Write each slice
+        for idx in range(20):
+            if -1 in low_res_offsets[idx]:
+                continue 
+            
+            t, b, l, r = [k*2**(8-level) for k in low_res_offsets[idx]]
+            height, width = size[0]-t-b, size[1]-l-r 
+
+            data = wkw_dataset.read(off = (l, t, idx), shape = [width, height, 1])
+            data = data[0, :, :, 0]
+            data = np.transpose(data, (1, 0))
+            if is_contour and level > 3: 
+                if initials == 'EB':
+                    mapped_img = np.array([[dic_EB[data[i][j]] for j in range(data.shape[1])] for i in range(data.shape[0])])
+                elif initials == 'JW': 
+                    mapped_img = np.array([[dic_JW[data[i][j]] for j in range(data.shape[1])] for i in range(data.shape[0])])
+                elif initials == 'JS': 
+                    mapped_img = np.array([[dic_JS[data[i][j]] for j in range(data.shape[1])] for i in range(data.shape[0])])
+                subdat = generate_mask(mapped_img)
+
+            else: 
+                subdat = data 
+            subdat_size = subdat.shape 
+            
+            print('Convert level', level, 'with shape', shape, 'and slice', idx, 'with size', subdat_size)
+            if max_load is None or (subdat_size[-2] < max_load and subdat_size[-1] < max_load):
+                array[0, idx, t: t+subdat_size[-2], l: l+subdat_size[-1]] = subdat[...]
+            else:
+                ni = ceildiv(subdat_size[-2], max_load)
+                nj = ceildiv(subdat_size[-1], max_load)
+                
+                for i in range(ni):
+                    for j in range(nj):
+                        print(f'\r{i+1}/{ni}, {j+1}/{nj}', end=' ')
+                        start_x, end_x = i*max_load, min((i+1)*max_load, subdat_size[-2])
+                        start_y, end_y = j*max_load, min((j+1)*max_load, subdat_size[-1])
+                        array[0, idx,  t + start_x: t + end_x, l + start_y: l + end_y] = subdat[start_x: end_x, start_y: end_y]
+                print('')
+                
+        
+    # Write OME-Zarr multiscale metadata
+    print('Write metadata')
```

Comment:
Do the annotation have the exact same shape as the annotated zarr file? I feel like we should just copy that info from the input zarr.